qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -39,9 +39,12 @@ def _get_accessible_raw_data(user):
     raw_data
     """
     d = {}
-    for sid in user.user_studies:
+    accessible_studies = user.user_studies.union(user.shared_studies)
+    for sid in accessible_studies:
         for rdid in Study(sid).raw_data():
-            d[int(rdid)] = Study(RawData(rdid).studies[-1]).title
+            rdid = int(rdid)
+            if rdid not in d:
+                d[rdid] = Study(RawData(rdid).studies[-1]).title
```

Comment:
Ok, since there is no reason to do this, I'm going to change this to go the more efficient way. I noticed this when I was reworking this part of the code, but I preferred to leave it as it was rather than add more and more changes to an already gigantic PR. Thanks for the discussion.